src/test-harness/SparkEthereumTests.sol

```diff
@@ -158,13 +160,26 @@ abstract contract SparkEthereumTests is SparklendTests {
                 // A payload is defined for this domain
                 // We verify the mainnet spell defines this payload correctly
                 address mainnetPayload = _getForeignPayloadFromMainnetSpell(chainId);
-                console.log("mainnetPayload", mainnetPayload);
-                console.log("payload", payload);
                 assertEq(mainnetPayload, payload, "Mainnet payload not matching deployed payload");
             }
         }
     }
 
+    function _checkStorageSlot(address target, uint256 limit) internal view {
```

Comment:
This won't work, you have to use vm.load for checking storage directly

Reply:
used vm.load